src/utils/lodf_calculations.jl

```diff
@@ -0,0 +1,11 @@
+function buildlodf(branches::Array{T}, nodes::Array{Bus}, dist_slack::Array{Float64}=[0.1] ) where {T<:Branch}
+    linecount = length(branches)
+    ptdf , a = PowerSystems.buildptdf(branches,nodes,dist_slack)
+    H = zeros(Float64,linecount,linecount)
+    H = gemm('N','N',ptdf,a)
+    h = diag(H)
+    h = reshape(h,(linecount,1))
+    LODF = H ./ (ones(Float64,linecount,linecount) - gemm('N','T',ones(Float64,linecount,1),h))
```

Comment:
Why there are two operations on `h`? seems inefficient to allocate it twice. The same applies to the element-wise division

Reply:
'''
    H = gemm('N','N',ptdf,a)
    h = reshape(diag(gemm('N','N',ptdf,a)),(linecount,1))
    LODF = gemm('N','N',ptdf,a) ./ (ones(Float64,linecount,linecount) - gemm('N','T',ones(Float64,linecount,1),h))
    LODF = LODF - Array(Diagonal(LODF)) - eye(linecount,linecount) 
'''
Is this what you want ?

Reply:
Yes and also why having the element-wise operation `H ./ (ones(Float64,linecount,linecount)` ?

Reply:
```
    H = gemm('N','N',ptdf,a)
    h = reshape(diag(gemm('N','N',ptdf,a)),(linecount,1))
    (Dem, dipiv, dinfo) = getrf!(eye(Float64,linecount,linecount) - Array(Diagonal(H)))
    LODF = gemm('N','N',H,getri!(Dem,dipiv))
    LODF = LODF - Array(Diagonal(LODF)) - eye(linecount,linecount) 
```
This works faster